packages/core/src/browser/view-container.ts

```diff
@@ -853,15 +873,33 @@ export class ViewContainerPart extends BaseWidget {
 
         const title = document.createElement('span');
         title.classList.add('label', 'noselect');
+
+        const subHeading = document.createElement('span');
+        subHeading.classList.add('subheading');
+
         const updateTitle = () => title.innerText = this.wrapped.title.label;
         const updateCaption = () => title.title = this.wrapped.title.caption || this.wrapped.title.label;
+        const updateSubHeading = () => {
+            if (DescriptionWidget.is(this.wrapped) && !this.collapsed) {
+                subHeading.innerText = !!this.wrapped.description ? this.wrapped.description : '';
+            } else {
+                subHeading.innerText = '';
+            }
```

Comment:
Nitpick: We can simplify this

```suggestion
            subHeading.innerText = DescriptionWidget.is(this.wrapped) && !this.collapsed && this.wrapped.description || '';
```

Reply:
👍